eng/Versions.props

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <MicrosoftWindowsCsWinRTPackageVersion>1.4.1</MicrosoftWindowsCsWinRTPackageVersion>
+    <MicrosoftWindowsCsWinRTPackageVersion>1.5.0-prerelease.220124.4</MicrosoftWindowsCsWinRTPackageVersion>
     <CsWinRTDependencyDotNetCoreSdkPackageVersion>5.0.404</CsWinRTDependencyDotNetCoreSdkPackageVersion>
-    <CsWinRTDependencyDotNetCoreRuntimePackageVersion>5.0.13</CsWinRTDependencyDotNetCoreRuntimePackageVersion>
-    <CsWinRTDependencyWindowsSdkPackagePackageVersion>10.0.18362.0</CsWinRTDependencyWindowsSdkPackagePackageVersion>
+    <CsWinRTDependencyDotNetCoreRuntimePackageVersion>5.0.11</CsWinRTDependencyDotNetCoreRuntimePackageVersion>
+    <CsWinRTDependencyWindowsSdkPackagePackageVersion>10.0.18362.22</CsWinRTDependencyWindowsSdkPackagePackageVersion>
```

Comment:
The uses of CsWinRTDependencyWindowsSdkPackagePackageVersion will need to be fixed.  It should be used as part of WindowsSdkPackageVersion and not as part of CSWinRTWindowsMetadata.

FYI @j0shuams